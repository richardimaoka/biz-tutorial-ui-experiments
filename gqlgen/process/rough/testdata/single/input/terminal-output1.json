{
  "step": "29af3a1b-af1c-4472-857f-ff55271dccc2",
  "phase": "CMD の動作理解",
  "comment": "",
  "type": "terminal output",
  "instruction": "[+] Building 16.0s (6/6) FINISHED\n => [internal] load build definition from Dockerfile.cmd1                                                                                                                                                   0.0s\n => => transferring dockerfile: 80B                                                                                                                                                                         0.0s\n => [internal] load .dockerignore                                                                                                                                                                           0.0s\n => => transferring context: 2B                                                                                                                                                                             0.0s\n => [internal] load metadata for docker.io/library/ubuntu:latest                                                                                                                                            3.3s\n => [auth] library/ubuntu:pull token for registry-1.docker.io                                                                                                                                               0.0s\n => [1/1] FROM docker.io/library/ubuntu@sha256:aabed3296a3d45cede1dc866a24476c4d7e093aa806263c27ddaadbdce3c1054                                                                                            12.5s\n => => resolve docker.io/library/ubuntu@sha256:aabed3296a3d45cede1dc866a24476c4d7e093aa806263c27ddaadbdce3c1054                                                                                             0.0s\n => => sha256:445a6a12be2be54b4da18d7c77d4a41bc4746bc422f1f4325a60ff4fc7ea2e5d 29.54MB / 29.54MB                                                                                                           11.2s\n => => sha256:aabed3296a3d45cede1dc866a24476c4d7e093aa806263c27ddaadbdce3c1054 1.13kB / 1.13kB                                                                                                              0.0s\n => => sha256:b492494d8e0113c4ad3fe4528a4b5ff89faa5331f7d52c5c138196f69ce176a6 424B / 424B                                                                                                                  0.0s\n => => sha256:c6b84b685f35f1a5d63661f5d4aa662ad9b7ee4f4b8c394c022f25023c907b65 2.30kB / 2.30kB                                                                                                              0.0s\n => => extracting sha256:445a6a12be2be54b4da18d7c77d4a41bc4746bc422f1f4325a60ff4fc7ea2e5d                                                                                                                   0.9s\n => exporting to image                                                                                                                                                                                      0.0s\n => => exporting layers                                                                                                                                                                                     0.0s\n => => writing image sha256:710d7aba73162de2e4f1a7759908ddc6164ce9ea2deb0c7eccee358fdc701c16                                                                                                                0.0s\n => => naming to docker.io/library/cmd1",
  "commit": "",
  "tooltip": ""
}
