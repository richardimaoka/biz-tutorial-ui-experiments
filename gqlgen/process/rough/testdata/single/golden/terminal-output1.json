[
  {
    "fromRoughStep": true,
    "subId": "terminalOutputStep",
    "parentStep": "29af3a1b-af1c-4472-857f-ff55271dccc2",
    "step": "",
    "focusColumn": "Terminal",
    "column1": "Terminal",
    "terminalType": "output",
    "terminalText": "[+] Building 16.0s (6/6) FINISHED\n =\u003e [internal] load build definition from Dockerfile.cmd1                                                                                                                                                   0.0s\n =\u003e =\u003e transferring dockerfile: 80B                                                                                                                                                                         0.0s\n =\u003e [internal] load .dockerignore                                                                                                                                                                           0.0s\n =\u003e =\u003e transferring context: 2B                                                                                                                                                                             0.0s\n =\u003e [internal] load metadata for docker.io/library/ubuntu:latest                                                                                                                                            3.3s\n =\u003e [auth] library/ubuntu:pull token for registry-1.docker.io                                                                                                                                               0.0s\n =\u003e [1/1] FROM docker.io/library/ubuntu@sha256:aabed3296a3d45cede1dc866a24476c4d7e093aa806263c27ddaadbdce3c1054                                                                                            12.5s\n =\u003e =\u003e resolve docker.io/library/ubuntu@sha256:aabed3296a3d45cede1dc866a24476c4d7e093aa806263c27ddaadbdce3c1054                                                                                             0.0s\n =\u003e =\u003e sha256:445a6a12be2be54b4da18d7c77d4a41bc4746bc422f1f4325a60ff4fc7ea2e5d 29.54MB / 29.54MB                                                                                                           11.2s\n =\u003e =\u003e sha256:aabed3296a3d45cede1dc866a24476c4d7e093aa806263c27ddaadbdce3c1054 1.13kB / 1.13kB                                                                                                              0.0s\n =\u003e =\u003e sha256:b492494d8e0113c4ad3fe4528a4b5ff89faa5331f7d52c5c138196f69ce176a6 424B / 424B                                                                                                                  0.0s\n =\u003e =\u003e sha256:c6b84b685f35f1a5d63661f5d4aa662ad9b7ee4f4b8c394c022f25023c907b65 2.30kB / 2.30kB                                                                                                              0.0s\n =\u003e =\u003e extracting sha256:445a6a12be2be54b4da18d7c77d4a41bc4746bc422f1f4325a60ff4fc7ea2e5d                                                                                                                   0.9s\n =\u003e exporting to image                                                                                                                                                                                      0.0s\n =\u003e =\u003e exporting layers                                                                                                                                                                                     0.0s\n =\u003e =\u003e writing image sha256:710d7aba73162de2e4f1a7759908ddc6164ce9ea2deb0c7eccee358fdc701c16                                                                                                                0.0s\n =\u003e =\u003e naming to docker.io/library/cmd1"
  }
]